controllers/removeMemberFromGroup.ts

```diff
@@ -0,0 +1,28 @@
+import { PrismaClient } from "@prisma/client";
+const prisma = new PrismaClient();
+
+export const removeMemberFromGroups = async (req: any, res: any) => {
+  try {
+    const body = req.body;
+    let result: any;
+    const count = await prisma.group.count();
+
+    if (count > 2) {
+      result = await prisma.group.deleteMany({
+        where: {
+          ref: {
+            in: req.body.ref,
+          },
+        },
+      });
```

Comment:
Absolute incorrect logic. Either you didn't understand the task, or haven't tested it at all.